yb-voyager/cmd/exportData.go

```diff
@@ -211,6 +210,29 @@ func checkDataDirs() {
 	}
 }
 
+func extractTableListFromString(flagTableList string) []string {
+	var finalTableList []string
+	if flagTableList != "" {
```

Comment:
Always arrange if conditions to reduce the required number of indentation levels. For example, in this particular case it is better to check if `flagTableList == ""` and then return empty slice which allows reducing indentation level of the current if block.

```
if flagTableList == "" {
    retur []string{}
}
// Now the remaining code is sure that `flagTableList` is non-empty.
tableList := utils.CsvStringToSlice(...)
if source.DBType != POSTGRESQL {
  return tableList
}
// Remaining code assumes postgresql source.
for _, table := range tableList {
  ...
}
return finalTableList
```
@see https://dave.cheney.net/practical-go/presentations/gophercon-israel.html#_return_early_rather_than_nesting_deeply
@see https://go.dev/doc/effective_go#if

Reply:
I have consciously tried to keep applying this in my code, but sometimes it just slips! Thanks for the catch again